.github/workflows/ci.yml

```
@@ -189,9 +176,18 @@ jobs:
       - name: Install latest rust toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
-          default: true
-          override: true
+          profile: minimal
+
+      - name: Cache Rust dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)